dbos/_sys_db.py

```diff
@@ -804,6 +817,31 @@ def record_operation_result(
                 raise DBOSWorkflowConflictIDError(result["workflow_uuid"])
             raise
 
+    def record_get_result(
+        self, resultWorkflowID: str, output: Optional[str], error: Optional[str]
```

Comment:
```suggestion
        self, result_workflow_id: str, output: Optional[str], error: Optional[str]
```
Consistent snake case? :)